bridges/WikipediaBridge.php

```diff
@@ -107,9 +107,18 @@ private function AddElementGeneric($element, $fullArticle){
 		// Clean the bottom of the featured article
 		$element->find('div', -1)->outertext = '';
 
+		// The title and URI of the article is best defined in an anchor containint the string '...' ('full article ...')
+		$target = $element->find('p/a', 0); // We'll use the first anchor as fallback
+		foreach($element->find('//a') as $anchor){
+			if(strpos($anchor->innertext, '...') !== false){
```

Comment:
Also, it should probably go something like `if !empty($item->title) && $element->find`